internal/server/openapi.go

```diff
@@ -548,8 +550,21 @@ func getFieldName(f reflect.StructField, parent reflect.Type) string {
 		if name == "-" {
 			return ""
 		}
+		validateFieldName(name)
 		return name
 	}
 
 	panic(fmt.Sprintf("field %q of struct %q must have a tag (json, form, or uri) with a name or '-'", f.Name, parent.Name()))
 }
+
+func validateFieldName(name string) {
+	// temporary allow list
+	switch name {
+	case "created_by", "unique_id":
```

Comment:
`created_by` is fixed by #3768

I think we'll be able to remove `uniqueID` soon, so it's probably not worth renaming that one. Let's remove it sometime in the next release or two.

Reply:
I removed `created_by` from this list, since #3768 has merged.